addons/sale/models/res_company.py

```diff
@@ -71,19 +71,25 @@ def action_open_sale_onboarding_sample_quotation(self):
             with the edi_invoice_template message loaded by default. """
         sample_sales_order = self._get_sample_sales_order()
         template = self.env.ref('sale.email_template_edi_sale', False)
-        action = self.env.ref('sale.action_open_sale_onboarding_sample_quotation').read()[0]
-        action['context'] = {
-            'default_res_id': sample_sales_order.id,
-            'default_use_template': bool(template),
-            'default_template_id': template and template.id or False,
-            'default_model': 'sale.order',
-            'default_composition_mode': 'comment',
-            'mark_so_as_sent': True,
-            'custom_layout': 'mail.mail_notification_paynow',
-            'proforma': self.env.context.get('proforma', False),
-            'force_email': True,
-            'mail_notify_author': True,
-        }
+
+        message_composer = self.env['mail.compose.message']\
+            .with_context(default_use_template=bool(template), mark_so_as_sent=True, custom_layout='mail.mail_notification_paynow', proforma=self.env.context.get('proforma', False), force_email=True, mail_notify_author=True)\
+            .create({'res_id': sample_sales_order.id, 'template_id': template and template.id or False, 'model': 'sale.order', 'composition_mode': 'comment'})
```

Comment:
More readable like this no ? =)
```suggestion
        message_composer = self.env['mail.compose.message']\
            .with_context(default_use_template=bool(template), mark_so_as_sent=True, custom_layout='mail.mail_notification_paynow', proforma=self.env.context.get('proforma', False), force_email=True, mail_notify_author=True)\
        message_composer = self.env['mail.compose.message'].with_context(
            default_use_template=bool(template),
            mark_so_as_sent=True,
            custom_layout='mail.mail_notification_paynow',
            proforma=self.env.context.get('proforma', False),
            force_email=True,
            mail_notify_author=True
        ).create({
            'res_id': sample_sales_order.id,
            'template_id': template.id and template.id or False,
            'model': 'sale.order',
            'composition_mode':
            'comment'})
```